src/commands/bonusBoon.ts

```diff
@@ -0,0 +1,50 @@
+import { find, reduce } from "lodash";
+import { actBonuses } from "../data/actBonus/all";
+import { Boon } from "../data/gods/god";
+import { Command } from "./command";
+import { abilityFormatter } from "./utils/formatters";
+
+type BonusBoonRecord = {
+  bonus: string;
+  boon: Boon;
+};
+
+const abilityMatchersMap: { [matcher: string]: BonusBoonRecord } = actBonuses
+  .map((god) =>
```

Comment:
Since most assist npcs aren't gods, maybe the noun here is something like "npc" or "assistant"?